pallets/manta-pay/src/lib.rs

```diff
@@ -233,7 +240,7 @@ pub mod pallet {
         ) -> DispatchResultWithPostInfo {
             let origin = ensure_signed(origin)?;
             FungibleLedger::<T>::transfer(
-                asset.id,
+                Self::id_from_field(asset.id).ok_or(Error::<T>::InvalidAssetId)?,
```

Comment:
I don't think so , but I haven't explicitly measured.
Actually I should probably change the benchmarking macro to also use native asset id, because of that, in case the native one has a worse case.

Reply:
https://github.com/Manta-Network/Manta/issues/899